DOCS-1664/1666/1712/1677: Rework manage machines R2D2 (#2576)
viamrobotics/docs: docs/fleet/machines.md

## Navigating the machine page

> Click on the **parts live** display at the top left corner of the page to open a menu with information about each {{< glossary_tooltip term_id="part" text="part" >}} of this machine.

Reviewer: I wasn't quite sure what a parts live display is, maybe status indicator makes more sense?
```suggestion
Next to the machine name, there is an indicator of the machine's status.
Click on the **status** dropdown to open a menu with information about each {{< glossary_tooltip term_id="part" text="part" >}} of your machine.
```
Also thre's no _this_, it's a random machine they'll be seeing.

Author: What I meant is that it says "0/2 parts live" or similar when not live. Definitely better to consider both views

> Once you connect to the `viam-server` instance on a part, this display includes its OS, Host, `viam-server` version, IP addresses, and what time it was last online:

Reviewer: it won't show last online if its live. Make sure you consider both views

![The machine page with part menu expanded](/fleet/app-usage/machine-page.png)

### Set up a new machine

<!-- TODO R2D2: might need screenshot and needs to be revisited once setup construction is finished -->

To connect to the `viam-server` instance on a part, follow the setup instructions.

> Click **View setup instructions** on the part dropdown menu.

Reviewer: so is that also where you'd move from one part to another? that feels like it might be confusing?

Author: No, i think that's the nested left hand menu where you can move between part cards. Maybe if I make it more clear where this one is?

> Click **View setup instructions** on the part dropdown menu.

Author: ```suggestion
Click **View setup instructions** on the part status dropdown menu in the upper left corner.
```

Once you connect to the `viam-server` instance on a part, this display includes its OS, Host, `viam-server` version, IP addresses, and what time it was last online or remote address (if live):

![The machine page with part menu expanded](/fleet/app-usage/machine-page.png)

### Set up a new machine

<!-- TODO R2D2: might need screenshot and needs to be revisited once setup construction is finished -->

To connect to the `viam-server` instance on a part, follow the setup instructions.
Click **View setup instructions** on the part status dropdown menu in the top left corner.

Select your system's architecture and select the version of the {{< glossary_tooltip term_id="RDK" text="RDK" >}} to use.
Then, follow the instructions on the page to connect and set up your machine.

{{% alert title="Tip" color="tip" %}}
The **Micro-RDK** is for [microcontrollers](/get-started/installation/prepare/microcontrollers/).

More in-depth information on installing `viam-server` can be found in our [Install Guide](/get-started/installation/#install-viam-server).
{{% /alert %}}

> Once all parts of your machine are set up and connected to the app, the **parts live** display at the top left corner of the page turns green.

Author: ```suggestion
Once all parts of your machine are set up and connected to the app, the part status display at the top left corner of the page turns green.
```
